introduction.md

```diff
@@ -1,2 +1,5 @@
 Description description description
-Git is best path evah! However Ruby Javascript HTML & CSS also rock
\ No newline at end of file
+Git is best path evah! However Ruby Javascript HTML & CSS also rock
+* Git
+* Ruby
+* Javascript
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?